Core/include/Acts/Utilities/PdgParticle.hpp

```diff
@@ -30,6 +30,7 @@ enum PdgParticle : int32_t {
   eAntiNeutron = -eNeutron,
   eProton = 2212,
   eAntiProton = -eProton,
+  lead = 1000822080
```

Comment:
```suggestion
  eLead = 1000822080
```

Reply:
what is the "e" for?

Reply:
I don't know but if I had to guess "enum". I just noticed it because all the other entries start with "e"

Reply:
okey! I imagine that it may help to distinguish eLead with Lead when someone would import all PdgParticles enum.